lib/id3c/cli/io/google.py

```diff
@@ -0,0 +1,75 @@
+"""
+Utilities for interacting with Google services.
+"""
+from enum import Enum
+from googleapiclient import discovery
+from googleapiclient.http import MediaIoBaseDownload
+from io import BytesIO
+from typing import Optional
+import re
+
+
+# See a full list of Google Drive MIME types at
+# https://developers.google.com/drive/api/v3/ref-export-formats
+
+class GoogleDriveExportFormat(Enum):
+    CSV = "CSV"
+    EXCEL = "Excel"
+    HTML = "HTML"
+    HTML_ZIPPED = "HTML zipped"
+    PDF = "PDF"
+    PLAIN_TEXT = "Plain text"
+    RICH_TEXT = "Rich text"
+    WORD = "Word"
+
+GOOGLE_FORMAT_MIME_MAP = {
+    GoogleDriveExportFormat.CSV: "text/csv",
+    GoogleDriveExportFormat.EXCEL: "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
+    GoogleDriveExportFormat.HTML: "text/html",
+    GoogleDriveExportFormat.HTML_ZIPPED: "application/zip",
+    GoogleDriveExportFormat.PDF: "application/pdf",
+    GoogleDriveExportFormat.PLAIN_TEXT: "text/plain",
+    GoogleDriveExportFormat.RICH_TEXT: "application/rtf",
+    GoogleDriveExportFormat.WORD: "application/vnd.openxmlformats-officedocument.wordprocessingml.document"
+}
+
+def export_file_from_google_drive(document_id: str, destination_format: GoogleDriveExportFormat) -> BytesIO:
+    """
+    From Google Drive export the specified document in the specified MIME format.
+    Requires that the GOOGLE_APPLICATION_CREDENTIALS environment variable be set
+    pointing to the service account credentials file and that the service account have
+    read access to the document being exported.
+
+    Not every type of document can be exported as every GoogleDriveExportFormat type.
+
+    If you export a Google Sheets spreadsheet as a CSV, the first tab will be exported.
```

Comment:
What does "the first tab will be exported" mean here?

Reply:
I updated the comment to
If you export a multi tabbed Google Sheets spreadsheet as a CSV, it's the first tab that will be exported.